animations/test/misc/animated_positioned_test.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/animated_positioned.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createAnimatedPositionedDemoScreen() => MaterialApp(
+      home: AnimatedPositionedDemo(),
+    );
+
+void main() {
+  group('AnimatedPositioned Tests', () {
+    testWidgets('Position of Button Changes on Tap', (tester) async {
+      await tester.pumpWidget(createAnimatedPositionedDemoScreen());
+
+      var button = find.byType(InkWell);
+
+      // get initial position of the widget
```

Comment:
nit: These comments are great, but wach should begin with a capital letter and end with a period.